### PR TITLE
add chart lint ci workflow

### LINKED ISSUE #4061: use [helm/chart-testing-action@v2.2.1](https://github.com/helm/chart-testing) to validate chart
use [helm/chart-testing-action@v2.2.1](https://github.com/helm/chart-testing) to validate chart
/help

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**

**What this PR does / why we need it**:
add chart lint ci workflow to automatically validate charts files
**Which issue(s) this PR fixes**:

Fixes #4061 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: .github/workflows/lint-chart.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+          architecture: x64
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.2.1
+        with:
+          version: v3.6.0
+
```
Comment (wackxu) [inline .github/workflows/lint-chart.yaml:44]: remove one newline

File: manifests/charts/cloudcore/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+description: The KubeEdge cloudcore component.
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
 version: 0.1.0
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
 appVersion: 1.9.1
```
Comment (wackxu) [inline manifests/charts/cloudcore/Chart.yaml:12]: do we need update this version since we have released version 1.11?

Comment (zhu733756) [reply manifests/charts/cloudcore/Chart.yaml:12]: Seems no need to update it every time. But once the chart have brought big changes, we had better upgrade it.

**Pull request closed without merging**